.mailmap

```diff
@@ -1182,6 +1182,8 @@ Pastafarianist <mr.pastafarianist@gmail.com>
 Patrick Lacasse <patrick.m.lacasse@gmail.com>
 Patrick Poitras <acebulf@gmail.com>
 Paul Mandel <paulmandel@google.com>
+Paul Scofield <cerebral.paul@gmail.com>
+Paul Scofield <Cerebral.Paul@gmail.com>
```

Comment:
There should not be two lines in .mailmap:
https://github.com/sympy/sympy/blob/7450b1a76230162fb2b05bfcde56c2be6095dfae/.mailmap#L129-L148